Two-body-scf-guess.py

```diff
@@ -0,0 +1,93 @@
+from wrap_pyscf import *
+import numpy as np
+
+mol = [("O", -0.167787, 1.645761, 0.108747),
+       ("H", 0.613411, 1.102620, 0.113724),
+       ("H", -0.093821, 2.209720, -0.643619),
+       ("O", 1.517569, -0.667424, -0.080674),
+       ("H", 1.989645, -1.098799, 0.612047),
+       ("H", 0.668397, -1.091798, -0.139744),
+       ("O", -1.350388, -0.964879, -0.092208),
+       ("H", -1.908991, -1.211298, 0.626207),
+       ("H", -1.263787, -0.018107, -0.055536)]
+
+bs = "aug-cc-pvdz"
+monomers = [mol[0:3], mol[3:6], mol[6:9]]
+dimers = [[mol[0:6], (mol[0:3]+mol[6:9])], [mol[0:6], mol[3:9]], [(mol[0:3]+mol[6:9]), mol[3:9]]]
+#dimers = [[(monomers[0], monomers[1]), (monomers[0], monomers[2]), (monomers[0], monomers[3])], 
+          #[(monomers[0], monomers[1]), (monomers[1], monomers[2]), (monomers[1], monomers[3])],
+          #[(monomers[0], monomers[2]), (monomers[1], monomers[2]), (monomers[2], monomers[3])],
+          #[(monomers[0], monomers[3]), (monomers[1], monomers[3]), (monomers[2], monomers[3])]]
+
+def make_monomer_dens(monomers, bs):
+    monomer_dens = []
+    for j in monomers:
+        j = density(j, bs)
```

Comment:
Overwriting your loop variable like this (especially with a completely different object; `j` goes from molecules instances to density instances) is an easy way to introduce subtle bugs. I strongly recommend avoiding this.